modules/articles/client/config/articles.client.routes.js

```diff
@@ -12,22 +12,57 @@ angular.module('articles').config(['$stateProvider',
       })
       .state('articles.list', {
         url: '',
-        templateUrl: 'modules/articles/client/views/list-articles.client.view.html'
+        templateUrl: 'modules/articles/client/views/list-articles.client.view.html',
+        controller: 'ArticlesListController',
+        resolve:{
+          articles: function (Articles, $stateParams, $state) {
```

Comment:
don't like having the articles list in the resolve, the service should still be a part of the controller.